### PR TITLE
add constructor, fromJson, fromDb, toJson, save, and update

### PR DESCRIPTION
**Pull request by eyvmal**
Modified domain/cohort.js with new functions:
- constructor
- fromJson
- toJson
- fromDb
- save
- update

### CODE REVIEW

File: src/domain/cohort.js [hunk 1]
```diff
@@ -1,27 +1,92 @@
...
   const createdCohort = await dbClient.cohort.create({
-    data: {}
+    data: {
+      cohortName,
+      startDate,
+      endDate
+    }
   })
 
   return new Cohort(createdCohort.id)
```
Comment (JHalvor) [inline src/domain/cohort.js:17]: shouldn't this return the whole createdCohort object?

Comment (JHalvor) [reply src/domain/cohort.js:17]: maybe be return Cohort.fromDb(createdCohort)?

**Review by JHalvor — CHANGES_REQUESTED** [review]
needs to be looked at

File: src/domain/cohort.js
Comment (JHalvor) [inline src/domain/cohort.js:1]: looks good!

**Review by JHalvor — APPROVED** [review]
Nice! Looks good

**Review by Brandsegg — APPROVED** [review]
Looks good!

**Pull request merged**